epoxy-sample/src/main/java/com/airbnb/epoxy/VerticalGridCardSpacingDecoration.java

```diff
@@ -45,6 +45,9 @@ public void getItemOffsets(Rect outRect, View view, RecyclerView parent, State s
       // Only item in row
       outRect.left = outerPadding;
       outRect.right = outerPadding;
+      // Assign top and bottom padding if there is just one item in the row
+      outRect.top = innerPadding;
```

Comment:
Sorry, I'm missing the need for this. Top and bottom padding shouldn't need to change if there is only one item vs many. Could you explain it? If you want to add top/bottom padding to the edit text layout you can set it directly on `model_edit_text.xml`. I think views should generally be able to have different vertical padding if they need it